ulmo/runs/SSL/v2/latents_extraction_v2.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import print_function
```

Comment:
Do we need a unique copy of this code for each version?

Just checking.

Reply:
Thanks for that. We do not need this file since we will use the ssl_v2 to train and evaluate the code. I will remove this one.